styles/components/dropdown.scss

```diff
@@ -66,15 +64,33 @@
     // 4px = allowance
   }
 
-  // Dropup
+  > .menu > .divider {
+    height: 1px;
+    margin: 5px 0px;
```

Comment:
Okay nice, but `.menu uses this: `padding: ($padding-x / 2) 0;`. I think we should use the same rule.